templates/python-tests/src/ingest/kafka_tests.py

```diff
@@ -0,0 +1,55 @@
+from pydantic import BaseModel
+from moose_lib import Key, OlapTable, OlapConfig, Stream, IngestApi
+from moose_lib.blocks import KafkaEngine
+from moose_lib.dmv2 import MaterializedView, MaterializedViewOptions, IngestConfigWithDestination
+
+
+class KafkaTestEvent(BaseModel):
+    event_id: Key[str]
+    user_id: str
+    event_type: str
+    amount: float
+    timestamp: int
+
+
+kafka_test_input_stream = Stream[KafkaTestEvent]("kafka_test_input")
+
+kafka_test_ingest_api = IngestApi[KafkaTestEvent](
+    "kafka-test",
+    IngestConfigWithDestination(destination=kafka_test_input_stream)
+)
+
+kafka_test_source_table = OlapTable[KafkaTestEvent](
+    "kafka_test_source",
+    OlapConfig(
+        engine=KafkaEngine(
+            broker_list="redpanda:9092",
+            topic_list="kafka_test_input",
```

Comment:
maybe don't write the same string twice
`Stream[KafkaTestEvent]("kafka_test_input")`